src/reanimated2/hook/useAnimatedGestureHandler.ts

```diff
@@ -1,28 +1,14 @@
 'use strict';
 import type {
-  __Context,
-  __WorkletFunction,
-  __NativeEvent,
-} from '../commonTypes';
-import type { DependencyList } from './commonTypes';
-import { useEvent } from './useEvent';
+  DependencyList,
+  NativeEventWrapper,
+  ReanimatedEvent,
+} from './commonTypes';
 import { useHandler } from './useHandler';
+import { useEvent } from './useEvent';
+import type { PanGestureHandlerGestureEvent as DefaultEvent } from 'react-native-gesture-handler';
```

Comment:
We should remove dependency to gesture-handler if it is possible

Reply:
ready